app/models/planned_disbursement.rb

```diff
@@ -14,11 +12,9 @@ class PlannedDisbursement < ApplicationRecord
     :value,
     :providing_organisation_name,
     :providing_organisation_type,
-    :receiving_organisation_name,
-    :receiving_organisation_type,
+    :providing_organisation_reference,
     :financial_quarter,
-    :financial_year,
-    :providing_organisation_reference
+    :financial_year
   validates :value, inclusion: {in: 0.01..99_999_999_999.00}
 
   def unknown_receiving_organisation_type?
```

Comment:
If we don't collect `receiving_organisation` anymore. Do we want to keep this method?

Reply:
Nevermind. I just saw you removed it on your last commit :-)